backend/azureblob/azureblob.go

```diff
@@ -370,16 +370,19 @@ func (o *Object) split() (container, containerPath string) {
 }
 
 // validateAccessTier checks if azureblob supports user supplied tier
+// by comparing tier to the azblob.AccessTier* constants in a case-insensitive
+// manner
 func validateAccessTier(tier string) bool {
-	switch tier {
-	case string(azblob.AccessTierHot),
-		string(azblob.AccessTierCool),
-		string(azblob.AccessTierArchive):
-		// valid cases
+
+	if strings.EqualFold(tier, string(azblob.AccessTierHot)) ||
```

Comment:
This can just be

```go
return strings.EqualFold(tier, string(azblob.AccessTierHot)) ||
    strings.EqualFold(tier, string(azblob.AccessTierCool)) ||
    strings.EqualFold(tier, string(azblob.AccessTierArchive))
```

I think